src/main/java/uk/gov/hmcts/cmc/claimstore/models/ClaimData.java

```diff
@@ -157,7 +169,7 @@ public TheirDetails getDefendant() {
 
     @JsonIgnore
     public Boolean isClaimantRepresented() {
-        return claimant.getRepresentative().isPresent();
+        return claimants.stream().flatMap(p -> Optionals.toStream(p.getRepresentative())).findFirst().isPresent();
```

Comment:
findAny?

Reply:
You can use `return claimants.stream().anyMatch(claimant -> claimant.getRepresentative().isPresent());` instead

Reply:
actioned